graphql_compiler/schema_generation/sqlalchemy/scalar_type_mapper.py

```diff
@@ -175,15 +176,26 @@
 )
 
 
-def try_get_graphql_scalar_type(column_name, column_type):
+def try_get_graphql_scalar_type(
+    column_name: str, column_type: TypeEngine
+) -> Optional[GraphQLScalarType]:
     """Return the matching GraphQLScalarType for the SQL datatype or None if none is found."""
-    maybe_graphql_type = SQL_CLASS_TO_GRAPHQL_TYPE.get(type(column_type), None)
-    if maybe_graphql_type is None:
-        # Trying to get the string representation of the SQLAlchemy JSON and ARRAY types
-        # will lead to an error. We therefore use repr instead.
+    # HACK(pmantica1): The code above implies that the GraphQL type is just a factor of the class
+    #                  of column_type. However, it is also a function of the attributes of
+    #                  column_type. We should probably refactor this file to better reflect that.
+    if isinstance(column_type, sqltypes.DateTime) and column_type.timezone:
         warnings.warn(
-            'Ignoring column "{}" with unsupported SQL datatype: {}'.format(
-                column_name, type(column_type).__name__
-            )
+            f'Ignoring column "{column_name}". Timezone aware datetime types are '
+            f"currently not supported."
```

Comment:
OrientDB datetimes can also be tz-aware, but I don't know how to tell if they are. Even if I knew how to extract this information, I think that it would probably require a non-trivial refactor to extract it and I am not sure this is worth effort at the moment.

Reply:
Agree, doesn't seem to be worth it.